Tried to start development but the npm dependencies are failing

Hi!
I tried to create my fork and run npm install but it looks like some dependencies are causing the process to fail.
I am trying both on WSL on a Windows 10 PC and on a PowerShell on the same Windows 10 PC.

On WSL:
node version: v14.2.0
npm version: 6.14.5

```pc-araujo-w10:~/documents/workspace/time-to-leave $ npm install

> time-to-leave@1.5.2-dev preinstall /mnt/c/Users/araujo/Documents/workspace/time-to-leave
> npx npm-force-resolutions

npx: installed 5 in 5.241s
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/normalize-path-2658b41a/package.json'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/normalize-path-2658b41a/index.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/normalize-path-2658b41a/LICENSE'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/normalize-path-2658b41a/README.md'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/es-abstract-4144c8f2/2018/IterableToList.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/jest-mock-a0677f1f/build-es5/index.js.map'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/es-abstract-4144c8f2/2015/IteratorClose.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/js-yaml-3358f4ef/dist/js-yaml.min.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/es-abstract-4144c8f2/2018/IteratorClose.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/stylelint-c1ca0c85/lib/rules/selector-combinator-blacklist/README.md'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/rxjs-effc252b/add/operator/groupBy.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/es-abstract-4144c8f2/2019/IteratorClose.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/stylelint-c1ca0c85/lib/rules/selector-combinator-space-after/README.md'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/rxjs-effc252b/internal/operators/groupBy.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/es-abstract-4144c8f2/2015/IteratorComplete.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/stylelint-c1ca0c85/lib/rules/selector-combinator-space-before/README.md'
.............
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/lodash-ff15867a/fp/_mapping.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/lodash-ff15867a/fp/_util.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/lodash-ff15867a/fp/add.js'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/jest/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/jest-watcher/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/jest-resolve-dependencies/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/@jest/reporters/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/@jest/core/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! Maximum call stack size exceeded
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/lodash-ff15867a/fp/after.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/lodash-ff15867a/fp/all.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/lodash-ff15867a/fp/allPass.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/lodash-ff15867a/fp/always.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/lodash-ff15867a/fp/any.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/lodash-ff15867a/fp/zipObjectDeep.js'
npm WARN tar ENOENT: no such file or directory, open '/mnt/c/Users/araujo/Documents/workspace/time-to-leave/node_modules/.staging/lodash-ff15867a/fp/zipWith.js'

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/araujo/.npm/_logs/2020-05-07T23_41_50_965Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -31,6 +31,7 @@ Build/Install in Windows or MacOS:
 ```bash
 git clone https://github.com/thamara/time-to-leave.git
 cd time-to-leave
+npm ci
 npm install
 ```
 

